DM-35667: Early DP0.2 schema fixups (mainly about times)

gpdf opened the pull request:
This is an omnibus fixup ticket for various small problems observed in the Felis schema for DP0.2.  Most of the issues are associated with time stamps, but not all.

JeremyMcCormick reviewed (changes requested):
The `mysql:datatype` lines could just be removed entirely, since `datatype: double` already maps to `DOUBLE` when using the mysql variant. I will be removing these redundant annotations in the future anyways across all of the schemas.

gpdf commented:
I thought changing them to mySQL DOUBLE first would make the paper trail of the eventual change clearer: instead of the history just looking like "we deleted the TIMESTAMP ones" it will look like "we fixed an error, and then we swept up all the redundant type overrides in one big bang".

JeremyMcCormick reviewed (approved):
After discussion, it was decided to leave the mysql annotations in this case and remove them later when this happens across all schemas. Since an MJD is not represented by a database `TIMESTAMP` type but a double precision number, this changes looks good.
